### PR TITLE
fix(curriculum): updated ES5 and ES6 class description

### LINKED ISSUE #47476: Confusing wording which should be changed
### Describe the Issue

The difference between ES5 and ES6 construction is the removal of keyword function. The constructor in ES6 or function constructor in ES5 is invoked by a new keyword.

The wording of the question says
> In ES5, we usually define a `constructor` function and use the new keyword to instantiate an object.

and then after code 
> The `class` syntax simply replaces the `constructor` function creation:

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function

### Your code

NA

### Expected behavior

In my experience, it should be
> The class syntax simply replaces the keyword `function` with `construction`.

### Screenshots

Screenshot 2022-09-09 at 10 20 04

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nayabatir1**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #47476

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ES6 provides a new syntax to create objects, using the <dfn>class</dfn> keyword.
 
 It should be noted that the `class` syntax is just syntax, and not a full-fledged class-based implementation of an object-oriented paradigm, unlike in languages such as Java, Python, Ruby, etc.
 
-In ES5, we usually define a `constructor` function and use the `new` keyword to instantiate an object.
+In ES5, an object can be created by defining a `constructor` function and using the `new` keyword to instantiate the object.
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:15]: The old ES5 example can be deleted, to simplify this and reduce the text

Comment (nayabatir1) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:15]: I think we should keep it for reference.
If you still want to remove it, let me know I'll do it.

Comment (jeremylt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:15]: I still want it removed, as discussed in the issue. 

It is best to follow the decisions from the issue when you make a contribution.

Comment (nayabatir1) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:15]: removed

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md [hunk 2]
```diff
@@ -12,24 +12,35 @@ ES6 provides a new syntax to create objects, using the <dfn>class</dfn> keyword.
...
+In ES6, a `class` declaration has a `constructor` method that is invoked with the `new` keyword. If `constructor` method is not explicitly defined, then it is implicitly defined with no arguments.
 
 ```js
+// Explicit constructor
 class SpaceShuttle {
   constructor(targetPlanet) {
     this.targetPlanet = targetPlanet;
   }
+  takeOff() {
+    console.log("To " + this.targetPlanet + "!") ;
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:26]:
```suggestion
    console.log("To " + this.targetPlanet + "!");
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md [hunk 3]
```diff
@@ -12,24 +12,35 @@ ES6 provides a new syntax to create objects, using the <dfn>class</dfn> keyword.
...
+// Implicit constructor 
+class Rocket {
+  launch() {
+    console.log("To the moon!") ;
+  }
+}
+
 const zeus = new SpaceShuttle('Jupiter');
+// prints To Jupiter! in console
+zeus.takeOff()
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:39]:
```suggestion
zeus.takeOff();
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md [hunk 4]
```diff
@@ -12,24 +12,35 @@ ES6 provides a new syntax to create objects, using the <dfn>class</dfn> keyword.
...
+  }
+}
+
 const zeus = new SpaceShuttle('Jupiter');
+// prints To Jupiter! in console
+zeus.takeOff() 
+
+const atlas = new Rocket();
+// prints To the moon! in console
+atlas.launch()
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:43]:
```suggestion
atlas.launch();
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md [hunk 5]
```diff
@@ -12,24 +12,35 @@ ES6 provides a new syntax to create objects, using the <dfn>class</dfn> keyword.
...
   }
+  takeOff() {
+    console.log("To " + this.targetPlanet + "!") ;
+  }
 }
+
+// Implicit constructor 
+class Rocket {
+  launch() {
+    console.log("To the moon!") ;
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:33]:
```suggestion
    console.log("To the moon!");
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md [hunk 6]
```diff
@@ -12,24 +12,35 @@ ES6 provides a new syntax to create objects, using the <dfn>class</dfn> keyword.
...
 
-```js
-var SpaceShuttle = function(targetPlanet){
-  this.targetPlanet = targetPlanet;
-}
-var zeus = new SpaceShuttle('Jupiter');
-```
-
-The `class` syntax simply replaces the `constructor` function creation:
+In ES6, a `class` declaration has a `constructor` method that is invoked with the `new` keyword. If `constructor` method is not explicitly defined, then it is implicitly defined with no arguments.
```
Comment (Sembauke) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:17]:
```suggestion
In ES6, a `class` declaration has a `constructor` method that is invoked with the `new` keyword. If the `constructor` method is not explicitly defined, then it is implicitly defined with no arguments.
```

**Review by Sembauke — DISMISSED** [review]

**Review by jeremylt — DISMISSED** [review]
Looks good to me once the suggestion is applied

**Review by jeremylt — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**